packages/frontend/preview/src/lib/map/map_preview.svelte

```
@@ -0,0 +1,125 @@
+    showZBands?: boolean;
+    zoom?: number;
+  };
+
+  let {
+    resolver,
+    mapTag,
+    width = 640,
+    height = 480,
+    showCollision = false,
+    showZBands = false,
+    zoom = 1,
```

Biome (2.5.7)

[error] This variable zoom is unused.

(lint/correctness/noUnusedVariables)